From 7205761d654636a6c1fad03a3ca40d38e253e192 Mon Sep 17 00:00:00 2001
From: Changqing Li <changqing.li@windriver.com>
Date: Fri, 25 Jan 2019 16:01:07 +0800
Subject: [PATCH] Makefile: fix problem that waf cannot found

tdb is subfolder of samba, but it can also build independently.
so both path need to be added into PATH

Upsteam-Status: Inappropriate [oe specific]

Fix reject by upstream since upsteam need to support both python2/3,
so drop add append PATH. refer link:
https://gitlab.com/samba-team/samba/merge_requests/209
https://gitlab.com/samba-team/samba/merge_requests/211

Signed-off-by: Changqing Li <changqing.li@windriver.com>
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 3e70146..64bb393 100644
--- a/Makefile
+++ b/Makefile
@@ -1,6 +1,6 @@
 # simple makefile wrapper to run waf
 
-WAF_BINARY=$(PYTHON) ../../buildtools/bin/waf
+WAF_BINARY=PATH=buildtools/bin:../../buildtools/bin:$$PATH waf
 WAF=PYTHONHASHSEED=1 WAF_MAKE=1 $(WAF_BINARY)
 
 all:
-- 
2.7.4

